src/weapons/Skin.java

```diff
@@ -3,97 +3,40 @@
 import java.util.ArrayList;
 import java.util.List;
 
+import lombok.Getter;
+import lombok.Setter;
+import lombok.EqualsAndHashCode;
+
+@Getter
+@Setter
+@EqualsAndHashCode
 public class Skin {
 	private String name;
 	private List<String> weapons = new ArrayList<String>();
 	private int id;
 	private String tag;
 	private String lang;
 	
-	public String getName() {
-		return name;
-	}
-	
-	public void setName(String name) {
-		this.name = name;
-	}
-	
-	public List<String> getWeapons() {
-		return weapons;
-	}
-	
-	public int getId() {
-		return id;
-	}
-	
-	public void setId(int id) {
-		this.id = id;
-	}
-	
-	public String getTag() {
-		return tag;
-	}
-	
-	public void setTag(String tag) {
-		this.tag = tag;
+	private String capitalize(String str) {
+		return str.substring(0, 1).toUpperCase() + str.substring(1);
 	}
 	
 	public String getLang() {
-		if (name.contains("phase1"))
-		{
-			return lang + " (Phase 1)";
-		}
-		else if (name.contains("phase2"))
-		{
-			return lang + " (Phase 2)";
-		}
-		else if (name.contains("phase3"))
-		{
-			return lang + " (Phase 3)";
-		}
-		else if (name.contains("phase4"))
-		{
-			return lang + " (Phase 4)";
-		}
-		else if (name.contains("emerald_marbleized"))
-		{
-			return lang + " (Emerald)";
-		}
-		else if (name.contains("ruby_marbleized"))
-		{
-			return lang + " (Ruby)";
-		}
-		else if (name.contains("sapphire_marbleized"))
-		{
-			return lang + " (Sapphire)";
-		}
-		else if (name.contains("blackpearl_marbleized"))
-		{
+		if (name.contains("phase)) {
+			return lang + " (" + capitalize(name.substring(0, 1)) + " " + name.charAt(5) + ")";
+		} else if (name.contains("marbleized") && !name.contains("blackpearl")) {
+			return lang + " (" + capitalize(name.replace("marbleized", "")) + ")";
+		} else if (name.contains("blackpearl_marbleized")) {
 			return lang + " (Black Pearl)";
 		}
 		return lang;
 	}
 	
-	public void setLang(String lang) {
-		this.lang = lang;
-	}
-	
 	public String weapons() {
 		StringBuilder sb = new StringBuilder();
 		for (String weapon : weapons) {
 			sb.append(weapon + ";");
 		}
 		return sb.toString();
 	}
-	
-	@Override
-	public boolean equals(Object obj) {
-		Skin skin = (Skin) obj;
-		return name.equals(skin.name);
-	}
-	
-	@Override
-	public int hashCode() {
-		return name.hashCode();
-	}
```

Comment:
It's been more than 4 years since I've last looked at this repository, but from looking at this method, I think I wanted to have Skin instances to be equal even if only name property matches. I assume Lombok generates a default equals and hash function using all the properties of the object (again, haven't used it for a long while now), which is not the same behavior with these overridden methods. Can you check?

Reply:
It's same. It's like that. Lombok does same thing.

Reply:
From https://projectlombok.org/features/EqualsAndHashCode:

> By default, it'll use all non-static, non-transient fields, but you can modify which fields are used (and even specify that the output of various methods is to be used) by marking type members with `@EqualsAndHashCode.Include` or `@EqualsAndHashCode.Exclude`. Alternatively, you can specify exactly which fields or methods you wish to be used by marking them with `@EqualsAndHashCode.Include` and using `@EqualsAndHashCode(onlyExplicitlyIncluded = true)`